integration-cli/docker_cli_port_test.go

```diff
@@ -221,3 +223,84 @@ func (s *DockerSuite) TestPortExposeHostBinding(c *check.C) {
 		c.Error("Port is still bound after the Container is removed")
 	}
 }
+
+func stopRemoveContainer(id string, c *check.C) {
+	runCmd := exec.Command(dockerBinary, "rm", "-f", id)
+	out, _, err := runCommandWithOutput(runCmd)
+	if err != nil {
+		c.Fatal(out, err)
+	}
+}
+
+func (s *DockerSuite) TestUnpublishedPortsInPsOutput(c *check.C) {
+	// Run busybox with command line expose (equivalent to EXPOSE in image's Dockerfile) for the following ports
+	port1 := 80
+	port2 := 443
+	expose1 := fmt.Sprintf("--expose=%d", port1)
+	expose2 := fmt.Sprintf("--expose=%d", port2)
```

Comment:
expose1 and expose2 are being used in three different places in this function.
I am not sure why we should recreate them each time.